### PR TITLE
net-fs/samba EAPI-7 bump, missing dep fix/cleanup

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: net-fs/samba/samba-4.13.3-r1.ebuild [hunk 1]
```diff
@@ -1,32 +1,47 @@
...
-EAPI=6
+EAPI=7
 
 PYTHON_COMPAT=( python3_{6..9} )
-PYTHON_REQ_USE='threads(+),xml(+)'
+PYTHON_REQ_USE="threads(+),xml(+)"
 inherit python-single-r1 waf-utils multilib-minimal linux-info systemd pam
 
-MY_PV="${PV/_rc/rc}"
-MY_P="${PN}-${MY_PV}"
```
Comment (Polynomial-C) [inline net-fs/samba/samba-4.13.3-r1.ebuild:11]: Please keep these

File: net-fs/samba/samba-4.13.3-r1.ebuild [hunk 2]
```diff
@@ -1,32 +1,47 @@
...
+DESCRIPTION="Samba Suite Version 4"
+HOMEPAGE="https://samba.org/"
 
+MY_P="${PN}-${PV/_rc/rc}"
 SRC_PATH="stable"
-[[ ${PV} = *_rc* ]] && SRC_PATH="rc"
-
+if [[ ${PV} = *_rc* ]]; then
+	SRC_PATH="rc"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
```
Comment (Polynomial-C) [inline net-fs/samba/samba-4.13.3-r1.ebuild:17]: You just inverted the logig here. If `${PV}` contains `_rc` we don't want to have any `KEYWORDS`.

**Review by Polynomial-C — CHANGES_REQUESTED** [review]

File: net-fs/samba/samba-4.13.3-r1.ebuild (hunk 2 above)
Comment (a17r) [reply net-fs/samba/samba-4.13.3-r1.ebuild:17]: ah, right.

File: net-fs/samba/samba-4.13.3-r1.ebuild (hunk 1 above)
Comment (a17r) [reply net-fs/samba/samba-4.13.3-r1.ebuild:11]: even if MY_PV is unused?

File: sys-libs/tevent/tevent-0.10.2.ebuild [hunk 3]
```diff
@@ -1,43 +1,48 @@
...
 	elibc_glibc? (
 		net-libs/libtirpc[${MULTILIB_USEDEP}]
 		|| (
 			net-libs/rpcsvc-proto
 			<sys-libs/glibc-2.26[rpc(+)]
 		)
 	)
-	${PYTHON_DEPS}
 "
-# build system does not work with python3
```
Comment (a17r) [inline sys-libs/tevent/tevent-0.10.2.ebuild:34]: that comment is probably also outdated ;)

Comment (Polynomial-C) [reply sys-libs/tevent/tevent-0.10.2.ebuild:34]: Yeah, drop it please.

**Review by Polynomial-C — APPROVED** [review]

**Pull request merged**